CHANGELOG.md

```diff
@@ -19,9 +19,129 @@ const expandWidget = new Expand({
 })
 ```
 
+### View Popup deprecations and changes
+In a continuous effort to optimize the performance and load time of the API, the Popup widget will no longer be bundled with the MapView and SceneView. The Popup widget receives new features regularly, such as when browsing related records. Each new feature added to the Popup widget increases the size of the widget and amount of built code that is sent to the web browser. For example, the Popup module, which is bundled with the MapView, represents around 50% of the size of the bundle.
+
+In this release, the Popup widget loading is deferred until the view is ready and finished updating, and will only be loaded if `view.popupEnabled` (previously `view.popup.autoOpenEnabled`) is `true`. The `view.popupEnabled` property is set to `true` by default, so if you aren't modifying any properties of the view's popup within your application, **there is no action needed**. 
+
+If modifying the view's popup properties or calling any of the popup methods, please expand the details below for breaking changes and the migration strategy.
```

Comment:
Should we remove the "expand" part and show all of it?  I think most people will want to read all of, so maybe not hide inside "expand"?